src/styles/card.scss

```
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
```

1. it's a good practice to move font styles to `html` tag
2. add `box-sizing: border-box` to `*` tag
3. reset default margin for `p` and `h2` tags
4. create `common.scss` file and move these styles there
```suggestion
* {
  box-sizing: border-box;
}

html {
  font-family: Roboto, sans-serif;
  font-weight: 400;
  font-size: 12px;
}

body,
h2,
p {
  margin: 0;
}
```

---

done